src/engine/engine_wi.c

```diff
@@ -133,13 +121,23 @@ static int _ocf_write_wi_core_write(struct ocf_request *req)
 	/* Get OCF request - increase reference counter */
 	ocf_req_get(req);
 
-	env_atomic_set(&req->req_remaining, 1); /* One core IO */
+	if (req->info.dirty_any) {
```

Comment:
This is a separate fix (not related to forward) that should be in a separate commit.

Reply:
#833